### PR TITLE
[Azure Search] Add Cognitive Skillset Operations

### PR DESCRIPTION
**Pull request by jeji1101**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by jeji1101** [comment]
The corresponding swagger spec PR is: Azure/azure-rest-api-specs#3386. Thanks.

**Comment by jeji1101** [comment]
Hi Bruce,

Thanks a lot. I will check with Victor tomorrow morning.

Sincerely,

Jennifer

Sent from my iPhone

On Jul 30, 2018, at 10:22 PM, Bruce Johnston <notifications@github.com<mailto:notifications@github.com>> wrote:


@jeji1101<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjeji1101&data=02%7C01%7Cjeji%40microsoft.com%7Cd5423ca4854b489df4f808d5f6a5a4dd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636686113657840572&sdata=nvYCTH%2Feh3MSr4wdTrwoOpmvFtmC9hKKEU7V%2FqhWKD4%3D&reserved=0> The code looks ok to me, other than a few minor documentation issues. I left some comments on the Swagger PR. I'll merge this once the last Swagger issues are resolved and you get signoff from another indexer/Grok SME (Eugene or Victor).

**Comment by dsgouda** [comment]
@jeji1101 Please join the Azure org on github [here](https://aka.ms/azuregithub) to trigger the CI builds

File: src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/SearchServiceClient.cs
Comment (jeji1101) [reply src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/SearchServiceClient.cs]: AutoRest add this. I didn't modify the generated code. Thanks for checking.

Comment (jeji1101) [reply src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/SearchServiceClient.cs]: I think it was added when I tried to debug the url issue. I have regenerated the generated code and the newly generated code removes the lines. I think the current change is correct. Thanks for checking.

File: src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/KeyPhraseExtractionSkill.cs
Comment (vl8163264128) [inline src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/KeyPhraseExtractionSkill.cs]: the maximum number of

**Review by vl8163264128 — APPROVED** [review]
models look good

**Review by bitsforthought — APPROVED** [review]

**Comment by brjohnstmsft** [comment]
@dsgouda One of the checks above didn't seem to run, so we can't merge this. Any idea how to get it to run?

**Comment by dsgouda** [comment]
@jeji1101 needs to join the Azure org as I pointed out to trigger the build

**Comment by jeji1101** [comment]
I used the link @dsgouda provided earlier to join Azure org around one or two hours ago. Thanks.

**Comment by dsgouda** [comment]
@azuresdkci Test this please

**Comment by stenvix** [comment]
Hi, when I can download this update using nuget?

**Comment by brjohnstmsft** [comment]
@stenvix This is one of two major features that we're adding to the next preview version of the Azure Search .NET SDK. The other one is support for [Complex Types](https://feedback.azure.com/forums/263029-azure-search/suggestions/6670910-modelling-complex-types-in-indexes). We're working towards releasing the next preview SDK in September.

**Pull request merged**
